### PR TITLE
BUG package libfreebl3.so cos7

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@isuruf @wolfv

File: recipe/meta.yaml [hunk 1]
```diff
@@ -134,7 +140,8 @@ about:
   home: http://sources.redhat.com/glibc/
   license: LGPL-2.0-or-later AND LGPL-2.0-or-later WITH exceptions AND GPL-2.0-or-later
```
Comment (isuruf) [inline recipe/meta.yaml:141]: Can you fix this too?

Comment (beckermr) [reply recipe/meta.yaml:141]: For sure but idk which exception to put in there.

Comment (isuruf) [reply recipe/meta.yaml:141]:
```suggestion
  license: LGPL-2.0-or-later AND LGPL-2.0-or-later WITH exceptions AND GPL-2.0-or-later AND MPL-2.0
```

Comment (beckermr) [reply recipe/meta.yaml:141]: Ohhh I thought you meant the "WITH exceptions" which is not proper SPDX.

**Pull request merged**